assets/js/components/notifications/dashboard-auth-alert.js

```diff
@@ -28,18 +28,26 @@ import Notification from '../notifications/notification';
 
 const DashboardAuthAlert = () => {
 	const { admin: { connectURL } } = global.googlesitekit;
-	const { currentAdminPage } = global.googlesitekit.admin;
-	const product = currentAdminPage
-		.replace( /googlesitekit|module|-/g, ' ' )
-		.replace( /(^\w{1})|(\s{1}\w{1})/g, ( match ) => match.toUpperCase() )
-		.trim();
-
+	const { unsatisfiedScopes } = global.googlesitekit.setup;
+	let message = '';
+	const missingScopes = [ ...new Set( unsatisfiedScopes.map( ( item ) => item.match( /https:\/\/www.googleapis.com\/auth\/([a-z]+)/ )[ 1 ] ) ) ];
+	const moduleNames = missingScopes.map( ( module ) => module.charAt( 0 ).toUpperCase() + module.slice( 1 ) );
```

Comment:
We should not be transforming the name from the scope like this as the matched string is only a "slug" at best. Instead we should be using `getModulesData()[ slug ].name` to get the localized name for the service (in a safer way of course to not assume the slug exists).

Also, this doesn't consider the few cases we have where the slug in the scope does not match exactly with one our module slugs. There are a few examples in the IB, but that's not an exhaustive list. We need to account for this mapping though.